src/GridItem.vue

```diff
@@ -664,7 +675,11 @@
             },
             // Helper for generating column width
             calcColWidth() {
-                var colWidth = (this.containerWidth - (this.margin[0] * (this.cols + 1))) / this.cols;
+                if (this.fixedWidth) {
+                    var colWidth = this.rowHeight;
```

Comment:
@gmsa this is where the colWidth is set to the same value as rowHeight. Which in turn makes it possible to create GridItems square. Then it's up to the user set same width and height of each GridItem.

Reply:
We had other solutions to this too, like setting the actual rowHeight the same way as the colWidth. If that is easier to understand we can update the PR. From my Point of view the important thing is have control over the colWidth too (doesn't have to be square in my case).